.github/workflows/windows.yml

```
@@ -0,0 +1,18 @@
+name: windows
+
+on: [push, pull_request]
+
+jobs:
+  # Build libamrex and all tutorials
+  tutorials:
+    name: MSVC C++17 w/o Fortran w/o MPI
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build & Install
+      run: |
+        mkdir build
+        cd build
+        cmake .. -DENABLE_FORTRAN=OFF -DENABLE_MPI=OFF -DCMAKE_CXX_STANDARD=17
+        cmake --build . --config Release
```

Let's try to build tutorials as well:
```suggestion
        cmake .. -DENABLE_TUTORIALS=ON -DENABLE_FORTRAN=OFF -DENABLE_MPI=OFF -DCMAKE_CXX_STANDARD=17
        cmake --build . --config Release --target tutorials
```